src/FormBuilder/resources/js/form-builder/src/blocks/fields/amount/inspector/index.tsx

```diff
@@ -14,8 +14,8 @@ import periodLookup from '../period-lookup';
 import RecurringDonationsPromo from '@givewp/form-builder/promos/recurring-donations';
 import {getFormBuilderWindowData} from '@givewp/form-builder/common/getWindowData';
 import {useCallback, useState} from '@wordpress/element';
-import Options from '@givewp/form-builder/components/OptionsPanel';
-import {OptionProps} from '@givewp/form-builder/components/OptionsPanel/types';
+import {OptionsPanel} from '@givewp/form-builder-library';
+import {OptionProps} from '@givewp/form-builder-library/build/OptionsPanel/types';
```

Comment:
FYI you can [import as type](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html#type-only-imports-and-export) so it gets excluded during runtime

```suggestion
import type {OptionProps} from '@givewp/form-builder-library/build/OptionsPanel/types';
```

Reply:
Resolved in 460514e85889540592879c6dc3612d84a90124af